test/e2e/performanceprofile/functests/0_config/config.go

```diff
@@ -127,6 +127,8 @@ var _ = Describe("[performance][config] Performance configuration", Ordered, fun
 		By("Waiting for MCP being updated")
 		mcps.WaitForCondition(performanceMCP.Name, mcv1.MachineConfigPoolUpdated, corev1.ConditionTrue)
 
+		Expect(testclient.Client.Get(context.TODO(), client.ObjectKeyFromObject(performanceProfile), performanceProfile))
+		Expect(fmt.Fprintf(GinkgoWriter, "Updated PerformanceProfile: %+v", performanceProfile))
```

Comment:
I'm still unconvinced we should fail if fmt.Fprintf fail, plus the `Expect` here seems wrong. Perhaps we should use https://pkg.go.dev/github.com/onsi/gomega/format instead of fprintf?

Reply:
Done